### PR TITLE
Improve ReorderableListView. When start Dragging, Extend list by adding last item

### PR DESCRIPTION
**Pull request by hmm-umm**
…ing last item>

### CODE REVIEW

**Comment by mugku** [comment]
Improve ReorderableListView. Improve ReorderableListView. When start Dragging, Extend list by adding last item.
change file : ..src/material/reorderable_list.dart

File: packages/flutter/lib/src/material/reorderable_list.dart [hunk 1]
```diff
@@ -59,13 +59,14 @@ class ReorderableListView extends StatefulWidget {
     @required this.onReorder,
     this.scrollDirection = Axis.vertical,
     this.padding,
+    this.canControlexpandedReorderList = false,
```
Comment (dnfield) [inline packages/flutter/lib/src/material/reorderable_list.dart]: nit: this parameter name seems a it confusing to me, and should probably have a capital `Expanded` anyway.

Also, please revert the formatting changes made in this file.

File: packages/flutter/lib/src/material/reorderable_list.dart [hunk 2]
```diff
@@ -83,6 +84,9 @@ class ReorderableListView extends StatefulWidget {
   /// The amount of space by which to inset the [children].
   final EdgeInsets padding;
 
+  /// When start Dragging, Control to Extend list by adding last item
```
Comment (dnfield) [inline packages/flutter/lib/src/material/reorderable_list.dart]: I'm not really sure what this means and it is not a complete sentence.  What is this actually doing?

All doc comments should start with a complete sentence ending with a `.`.

**Review by dnfield — CHANGES_REQUESTED** [review]
This needs tests.  It also could use some clarification about what's actually happening here for docs.  Formatting changes should be reverted.

**Pull request closed without merging**